py/dml/messages.py

```diff
@@ -1895,6 +1895,44 @@ def __init__(self, site, old, new, maybe_intended):
                       if maybe_intended else '')
         DMLWarning.__init__(self, site, old, new, suggestion)
 
+class WLGROUPSASLEV(DMLWarning):
+    """
+    The specified log level of a `log` looks as though you meant to specify the
+    log groups instead. For example:
+    ```
+    log info, some_log_group: ...;
+    log info, (some_log_group | another_log_group), 1: ...;
+    ```
+    If you indeed wanted to specify the log groups, make sure to also
+    explicitly specify the log level, and specify it first.
+    ```
+    log info, 1, some_log_group: ...;
+    log info, 1, (some_log_group | another_log_group): ...;
+    ```
+    """
+    fmt = ("the specified log level is likely intended to instead be a log "
+           + "groups specification. "
+           + "In order to specify log groups, the log level must be "
+           + "(explicitly) specified beforehand.")
```

Comment:
You should perhaps mention "disabled by default when SIMICS_API <= 6"

Reply:
In the docstring, that is? Having it be in the error message makes no sense.